src/System.Management.Automation/engine/hostifaces/History.cs

```diff
@@ -195,54 +147,31 @@ internal void SetEndTime(DateTime endTime)
         /// <param name="command"></param>
         internal void SetCommand(string command)
         {
-            _cmdline = command;
+            CommandLine = command;
         }
 
-        #endregion internal
-
-        #region private
-
         /// <summary>
         /// Id of the pipeline corresponding to this history entry
         /// </summary>
         private long _pipelineId;
 
         /// <summary>
-        /// Id of the history entry
-        /// </summary>
-        private long _id;
-
-        /// <summary>
-        /// CommandLine string
-        /// </summary>
-        private string _cmdline;
-
-        /// <summary>
-        /// ExecutionStatus of execution
-        /// </summary>
-        private PipelineState _status;
-
-        /// <summary>
-        /// Start time of execution
+        /// Gets a string representation of the execution time
         /// </summary>
-        private DateTime _startTime;
-
-        ///
-        ///End time of execution
-        ///
-        private DateTime _endTime;
-
-
-
-        /// <summary>
-        /// Flag indicating an entry is present/cleared
-        /// </summary>
-
-        private bool _cleared = false;
-
-        #endregion private
-
-        #region ICloneable Members
+        /// <returns></returns>
+        public string GetDurationString()
```

Comment:
Minor comment - should we follow `ToString()` and use `ToDurationString()` for public?

Reply:
Yes, or `Tostring(string)`. Thought about that a bit also, but never reach any real conclusion.